tests/test_cookiecutter_django_ida.py

```diff
@@ -95,10 +95,12 @@ def test_travis(options_baked):
 def test_upgrade(options_baked):
     """Make sure the upgrade target works"""
     try:
+        new_env = os.environ.copy()
```

Comment:
What was the role of this change? I wonder if it's inheriting the tox virtualenv and messing up the pip stuff.